therr-services/maps-service/src/api/aggregations.ts

```diff
@@ -1,3 +1,8 @@
+const getMetricsByName = (metrics, name) => {
+    const filteredMetrics = metrics.filter(metric => metric.name === name);
+    return filteredMetrics;
+};
+
```

Comment:
I think it would be more dynamic and efficient if we could group all metrics given a set of names. We can also do this as O(n) by first creating a bucket for each metric name, then iterating through the metrics just once and pushing each item to the respective bucket